nimbus/src/persistence.rs

```diff
@@ -22,13 +22,23 @@ const DB_VERSION: u16 = 1; // Increment and implement a DB migration in `maybe_u
 // Select the LMDB-powered storage backend when the feature is not activated.
 #[cfg(not(feature = "rkv-safe-mode"))]
 mod backend {
+    use rkv::backend::{
+        Lmdb, LmdbDatabase, LmdbEnvironment, LmdbRoCursor, LmdbRoTransaction, LmdbRwTransaction,
+    };
     use std::path::Path;
-    //use rkv::Readable;
-    use rkv::backend::{Lmdb, LmdbDatabase, LmdbEnvironment, LmdbRwTransaction};
 
     pub type Rkv = rkv::Rkv<LmdbEnvironment>;
     pub type RkvSingleStore = rkv::SingleStore<LmdbDatabase>;
+    pub type Reader<'t> = rkv::Reader<LmdbRoTransaction<'t>>;
     pub type Writer<'t> = rkv::Writer<LmdbRwTransaction<'t>>;
+    pub trait Readable<'r>:
+        rkv::Readable<'r, Database = LmdbDatabase, RoCursor = LmdbRoCursor<'r>>
+    {
+    }
+    impl<'r, T: rkv::Readable<'r, Database = LmdbDatabase, RoCursor = LmdbRoCursor<'r>>>
+        Readable<'r> for T
+    {
+    }
```

Comment:
Trait aliases are experimental, this is a hacky little workaround to simulate importing and specializing the `Readable` trait.